### ISSUE TITLE
Hardware Transcoding not working on AMD APU

### ISSUE THREAD

**Issue by Sgt-Ogre** [AlexxIT/go2rtc#188]
I could have my configuration wrong, but when i start to stream these video feeds it is definitely using the CPU for transcoding as it begins to use 75% of the CPU immediately. Is there a config option i need to enable in the YAML for it to use the GPU for 
transcoding? I have the go2rtc Master Hardware container installed.

I also see that Frigate 12.0 has go2rtc built in. Would it be worth it to port this config over to that? Would i be able to use Hardware Transcoding?

 Here is my config:

log:
  level: info  # default level
  api: debug
  rtsp: debug
  streams: debug
  webrtc: debug

api:
    listen: ":1984"
    
rtsp:
    listen: ":8554"

streams:
  
     doorbell_cam: rtsp://USERNAME:PASSWORD@IP:554/cam/realmonitor?channel=1&subtype=0&unicast=true
     driveway_cam: ffmpeg:rtsp://USERNAME:PASSWORD@IP:554/Streaming/Channels/1#video=h264#hardware
     sideyard_cam: ffmpeg:rtsp://USERNAME:PASSWORD@IP:554/Streaming/Channels/1#video=h264#hardware
     backyard_cam: ffmpeg:rtsp://USERNAME:PASSWORD@IP:554/#video=h264#hardware

**Comment by NickM-27** [comment]
You should use `radeontop` to confirm if the GPU is being used. AMD GPU works fine for me.

**Comment by Sgt-Ogre** [comment]
I am using it as a Home Assistant Add-On. Any idea on how to check on GPU?

**Comment by AlexxIT** [comment]
Yes. It's better to try with Frigate 12. Because they have experience with this hardware, and I don't.
If your GPU will work with Frigate 12, we can continue to search solution

**Comment by NickM-27** [comment]
> I am using it as a Home Assistant Add-On. Any idea on how to check on GPU usage inside of Home Assistant?

the amd driver is not supported for addon installs, so this failure is expected
